components/script/dom/htmlinputelement.rs

```diff
@@ -2291,6 +2292,21 @@ impl HTMLInputElement {
         let target = self.upcast::<EventTarget>();
 
         if self.Multiple() {
+            // When using WebDriver command element send keys,
+            // we are expected to append the files to the existing filelist.
+            if append {
```

Comment:
```suggestion
            if pref!(dom_testing_html_input_element_select_files_enabled) {
```

This suggestion is inspired by 291b42f6e972b2b46d2ff774404adebd53b51676 and is not ideal. I am thinking about creating another entry in `Preferences` which also does not seem ideal..

Another option is to revert https://github.com/servo/servo/issues/37843

What would you suggest? cc @jdm

Reply:
Hmm, my implementation is similar to chromium https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/chromedriver/chrome/web_view_impl.cc;l=1910-1934;bpv=0;bpt=1. With your approach, that means that file input through user interaction will also be appended when we open the webdriver port?

Reply:
I see your points. If we enable webdriver port but use it interactively, the interactive behaviour is also changed.

Then it is the same situation for 291b42f6e972b2b46d2ff774404adebd53b51676. Personally IDM as it makes things simpler.
But I'll defer to @jdm to decide.

Reply:
For now, I think we should still apply the suggestion so that we don't need to revert many things. Just use preference `dom_testing_html_input_element_select_files_enabled` for now to keep things simple.